### PR TITLE
Form branch

### PR DESCRIPTION
**Pull request by Nuga0909**
1.  Add form to html file
2. Add styling to form elements
3. Add validation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,179 +1,254 @@
...
+          placeholder="Email address"
+          name="Email address"
+          required
+        />
+        <textarea
+          name="Message"
+          id="textarea"
+          cols="50"
+          rows="10"
+          placeholder="Enter text here"
```
Comment (Nyame-Wolf) [inline index.html:248]: Kindly add the maxlength property to your form as required for HTML validation in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)
Screenshot 2022-11-01 at 14 57 53

File: index.html [hunk 2]
```diff
@@ -1,179 +1,254 @@
...
+        ></textarea>
+        <button type="submit" id="form-button">Get in touch</button>
+      </form>
     </section>
```
Comment (Nyame-Wolf) [inline index.html:254]: Kindly add a footer section with a horizontal line and social media icons as in the figma design. Its a requirement that you "must stick to the design as much as possible (e.g., font, colors, images, text, margins) using the templates in Figma as per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)
Screenshot 2022-11-02 at 10 48 11

File: index.html [hunk 3]
```diff
@@ -1,179 +1,254 @@
...
+          required
+        />
+        <textarea
+          name="Message"
+          id="textarea"
+          cols="50"
+          rows="10"
+          placeholder="Enter text here"
+        ></textarea>
+        <button type="submit" id="form-button">Get in touch</button>
```
Comment (Nyame-Wolf) [inline index.html:251]: Kindly center the button to match the figma design. Its a requirement that you match the figma design as much as possible

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @Nuga0909  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Form is being accepted
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by sajid-munawar — APPROVED** [review]
### Hi @Nuga0909 , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**